cmd/thetool/validate.go

```diff
@@ -37,6 +49,33 @@ func validateEventContent(ev []Event, p *Preset, countries []string) error {
 			return fmt.Errorf("the Key should not be in the input file")
 		}
 
+		for l, t := range ev[i].Title {
+			if err := textValidator(t, fmt.Sprintf("%s -> title -> %s", ev[i].PartialKey, l)); err != nil {
+				return err
+			}
+		}
+
+		for l, t := range ev[i].Description {
+			if err := textValidator(t, fmt.Sprintf("%s -> description -> %s", ev[i].PartialKey, l)); err != nil {
```

Comment:
you call `textValidator` like this in all calls. why not pass the values in a cleaner way to the function? something like this 

```go
func textValidator(text, partial, language, type string)
```